src/main/webapp/js/finscholar/commonlistview/commonlistview.soy

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{namespace finscholar.commonlistview.templates}
+
+{template .commonlistview}
+  {@param pageIndex : string}
+  <table class="mdl-data-table mdl-js-data-table 
+      mdl-data-table--selectable mdl-shadow--2dp">
+    <thead>
+      {if $pageIndex == '1'}
+        {call .scholarshiplistheader /}
+      {else}
+        {call .collegelistheader /}
+      {/if}
+    </thead>
+    <tbody id="table-body">
+    </tbody>
+  </table>
+  <div id="status">
+    {call .endoflist /}
+  </div>
+{/template}
+
+{template .scholarshiplistheader}
```

Comment:
Can you group the scholarship templates together to make this more readable?

Reply:
I made a common item template for college and scholarship, so we only have one scholarship template now.